### PR TITLE
Updates backbone

### PR DESCRIPTION
**Pull request by javierarce**
- Updates backbone
- Updates underscore

Manually tested the examples

### CODE REVIEW

File: test/spec/core/view.spec.js [hunk 1]
```diff
@@ -42,9 +42,9 @@ describe("core.view", function() {
 
   it("clean should unbind all events", function() {
       view.bind('meh', function(){});
-      expect(_.size(view._callbacks)).toEqual(1);
```
Comment (javierarce) [inline test/spec/core/view.spec.js]: I'm not sure about this change

Comment (javierarce) [inline test/spec/core/view.spec.js]: I'm not sure about the implications of this change.

**Comment by javierarce** [comment]
@xavijam, @alonsogarciapablo, @javisantana: please, have a look.

**Comment by javisantana** [comment]
:_)

File: vendor/backbone.js [hunk 2]
```diff
@@ -1,1431 +1,2 @@
...
...;for(var p=0;p<t.length;p++){g=t[p];var m=this.get(g);if(m){if(l&&g!==m){var _=this._isModel(g)?g.attributes:g;if(e.parse)_=m.parse(_,e);m.set(_,e);if(d&&!f)f=m.hasChanged(v)}if(!h[m.cid]){h[m.cid]=true;s.push(m)}t[p]=m}else if(u){g=t[p]=this._prepareModel(g,e);if(g){a.push(g);this._addReference(g,e);h[g.cid]=true;s.push(g)}}}if(c){for(p=0;p<this.length;p++){g=this.models[p];if(!h[g.cid])o.push(g)}if(o.length)this._removeModels(o,e)}var y=false;var b=!d&&u&&c;if(s.length&&b){y=this.length!=s.length||i.some(this.models,function(t,e){return t!==s[e]});this.models.length=0;k(this.models,s,0);this.length=this.models.length}else if(a.length){if(d)f=true;k(this.models,a,n==null?this.length:n);this.length=this.models.length}if(f)this.sort({silent:true});if(!e.silent){for(p=0;p<a.length;p++){if(n!=null)e.index=n+p;g=a[p];g.trigger("add",g,this,e)}if(f||y)this.trigger("sort",this,e);if(a.length||o.length)this.trigger("update",this,e)}return r?t[0]:t},reset:function(t,e){e=e?i.clone(e):{};for(var r=0;r<this.models.length;r++){this._removeReference(this.models[r],e)}e.previousModels=this.models;this._reset();t=this.add(t,i.extend({silent:true},e));if(!e.silent)this.trigger("reset",this,e);ret
```
Comment (viddo) [inline vendor/backbone.js]: Wouldn't it help to have the .map file added too? (same thing for the underscore file).

**Comment by javisantana** [comment]
BTW I'd use non minified vendor files, at the end we are going to do the minification in our build process

**Comment by viddo** [comment]
IMHO it would help to have the backbone version documented somewhere suitable. The underscore lib contains the version in an inlined comment.

**Comment by alonsogarciapablo** [comment]
:+1:

**Comment by xavijam** [comment]
\o/

**Comment by javierarce** [comment]
@viddo: since now we are going to use a non-minified version of backbone, we'll have the version in an inlined comment, just like underscore. Would that be OK or do you think we should find a better place?

**Comment by viddo** [comment]
@javierarce that's enough for me :smile: :+1:

**Comment by javierarce** [comment]
Perfect then :)

**Pull request merged**